addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -58,6 +58,43 @@ var TranslatableFieldMixin = {
     },
 };
 
+var CopyClipboard = {
```

Comment:
should be named CopyClipboardMixin. This is more explicit and easier to understand.